build.tf

```diff
@@ -0,0 +1,40 @@
+resource "digitalocean_droplet" "node_builder" {
+    image = "ubuntu-20-04-x64"
+    name = "${terraform.workspace}-safe-node-builder"
+    region = "lon1"
+    size = "s-8vcpu-16gb"
+    private_networking = true
+    ssh_keys = var.ssh_keys
+
+    connection {
+        host = self.ipv4_address
+        user = "root"
+        type = "ssh"
+        timeout = "10m"
+        private_key = file(var.pvt_key)
+    }
+
+    provisioner "remote-exec" {
+        inline = [
+            "git clone https://github.com/maidsafe/safe_network --depth 1 -q",
+            "cd safe_network",
+            "apt -qq update",
+            "sleep 50",
```

Comment:
why?

Reply:
Yeah I need to find a way around that. There is a DPKG lock that occurs during apt update and running the install command immediately after was throwing some errors. This was a temp fix. I'll find a graceful solution for this.